### DISCUSSION TITLE
Uygulama

### DISCUSSION THREAD

**Discussion by Ecenurcelik** [IBAX-io/go-ibax#1715] [Ideas]
İnsanlar bu siteye ve vaat ettiklerine çok inançlılar . Benim ise sizden beklediğim hatta herkesin beklentisi app store ve olay store da çok daha afilli ve hoş görünen bir uygulamadır . Tavsiyem bu yöndedir .

**Comment by Bulent3407** [comment]
Haklısınız bencede çok iyi olur👍
